lib/logstash/inputs/twitter.rb

```diff
@@ -145,6 +159,10 @@ def run(queue)
       @logger.warn("Twitter too many requests error, sleeping for #{sleep_for}s")
       Stud.stoppable_sleep(sleep_for) { stop? }
       retry
+    rescue IOError => e
+      @logger.info("Twitter error: #{e.message}, retry in #{steam_eof_wait_duration}s")
```

Comment:
maybe `@logger.error` here? 

```suggestion
      @logger.error("Socket error, retrying in #{steam_eof_wait_duration}s", :error => e.message)
```

Reply:
I chose INFO because it is not an error per se. When the user sees the ERROR log line they panic as if there is something that they need to do/fix. The message can be more INFO and not ERROR in its wording.

Reply:
maybe a good compromise with warn? :D Since that the ocurrence of this event means that there's no tweet ingestion I think we should do > info